src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -0,0 +1,164 @@
+Compat issues with assembly mscorlib:
```

Comment:
These look to me like we're not passing in the right set of seeds.  Shouldn't these be handled by the OOB implementations / partial facades?